metadata.mk

```diff
@@ -3,10 +3,10 @@
 #################################################################################################
 
 # The version of github.com/projectcalico/go-build to use.
-GO_BUILD_VER = v0.85
+GO_BUILD_VER = v0.88
 
 # Version of Kubernetes to use for tests.
-K8S_VERSION     = v1.24.7
+K8S_VERSION     = v1.26.6
```

Comment:
You may want to keep this as-is and update `KUBECTL_VERSION` to v1.26.8 instead.